go/src/koding/klient/machine/mount/index/index.go

```diff
@@ -12,6 +12,9 @@ import (
 	"github.com/djherbis/times"
 )
 
+// Version stores current version of index.
+const Version = 1
```

Comment:
Is it going to be manually bumped or set during build with ldflags? If the latter, then it should be var.